File: Java&Kotlin/src/main/java/baekjoon/bronze/b3/FBI.kt
package baekjoon.bronze.b3

fun main() {
    var cnt = 0
    repeat(5) {
        if (readln().contains("FBI")) {
            cnt++
            print("${it + 1} ")
        }
    }
    if (cnt == 0) print("HE GOT AWAY!")
}

File: Python/baekjoon/bronze/b3/FBI.py
l = [input() for _ in range(5)]

t = ''
for i in range(len(l)):
    s = l[i]

    if s.find('FBI') >= 0:
        t += str(i + 1) + ' '

t = t.strip()

print('HE GOT AWAY!' if t == '' else t)
